docs/tutorials/embassy/index.md

```diff
@@ -66,7 +66,10 @@ Options:
   -s, --serial   Connect to serial after deploy
   -h, --help     Print help
 ```
-
+:::info Before installing elf2uf2-rs, you might need an additional package, called libudev. You can get it by running the following in your terminal. :::
+```shell
+sudo apt-get install libudev-dev
+```
```

Comment:
This is only valid for Linux, please move it to the Linux part.

Reply:
Moved it, sorry for the inconvenience